src/components/Form/TextArea.tsx

```diff
@@ -43,15 +37,15 @@ function TextArea({
   placeholder,
   onFocus,
   onBlur,
+  value,
   ...otherProps
 }: TextAreaProps) {
   const { disabled } = otherProps
   variant = disabled ? 'disabled' : variant
   const isDisabled = variant === 'disabled'
   const { input, text } = inputVariants[variant]
   const [focused, setFocused] = useState(false)
-  const textareaRef = useRef<HTMLTextAreaElement>(null)
-  const isLabelScalded = !label || focused || textareaRef.current?.value.length
+  const isLabelScalded = !label || focused || value?.toLocaleString().length
```

Comment:
I think this way it will take it as undefined if the value is not passed to it explicitly in the component props.

Reply:
You are right @AntonioFeregrino , make a modification by adding the value of the ref. Thanks!